build-support/functions/10-util.sh

```diff
@@ -935,7 +935,7 @@ function shasum_directory {
       return 1
    fi
 
-   local ui_version=$(sed -n ${SED_EXT} -e 's/.*CONSUL_VERSION%22%3A%22([^%]*)%22%2C%22.*/\1/p' < "$1") || return 1
+   local ui_version="$(urldecode "$(grep '<meta name="consul-ui/config/environment' "$1" | grep -o 'content="[^"]*"' | cut -d'"' -f2)" | jq -r .CONSUL_VERSION)" || return 1
    echo "$ui_version"
```

Comment:
Not massively sure about using `jq` for this? It's not used anywhere else in the build scripts that I can see? What do others think? Would it be easier for me to just have a look at printing the version information out elsewhere in the HTML source, formatted how we want it? (That urlencoded information is how ember passes configuration information from node world to browser world, so it needs to stay like that)